src/main.cpp

```diff
@@ -16,18 +22,84 @@ class Command : public TgBot::BotCommand {
     }
 };
 
+bool HandleUser(size_t id, std::unordered_map<size_t, size_t>& users) {
+    ++users[id];
+    return users[id] > kMaxQueries;
+}
+
+std::vector<std::string> GetArguments(std::string message) {
+    std::vector<std::string> tokens;
+    boost::split(tokens, message, boost::is_any_of(" "));
+    return tokens;
+}
+
+std::vector<std::string> ReadDDLQueries(std::string filename) {
```

Comment:
Logic can be extracted to function that takes an iostream for parsing, and this function just simply open the file and call that function.

Reply:
done